src/test/java/cloud/fogbow/ras/core/plugins/interoperability/azure/network/sdk/AzureNetworkSDKTest.java

```diff
@@ -0,0 +1,93 @@
+package cloud.fogbow.ras.core.plugins.interoperability.azure.network.sdk;
+
+import cloud.fogbow.common.exceptions.UnexpectedException;
+import com.microsoft.azure.management.Azure;
+import com.microsoft.azure.management.network.NetworkInterface;
+import com.microsoft.azure.management.network.NetworkInterfaces;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import java.util.Optional;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({AzureNetworkSDK.class})
+public class AzureNetworkSDKTest {
+
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
+
+    // test case: When calling the getNetworkInterface method and finds a network interface,
+    // it must verify if It returns a Optional with a network interface.
+    @Test
+    public void testGetNetworkInterfaceSuccessfullyWhenFindNetworkInterface() throws Exception {
+        // set up
+        Azure azure = null;
+        String networkInterfaceId = "networkInterfaceId";
+        NetworkInterfaces networkInterfacesObject = Mockito.mock(NetworkInterfaces.class);
+        NetworkInterface networkInterfaceExpected = Mockito.mock(NetworkInterface.class);
+        Mockito.when(networkInterfacesObject.getById(Mockito.eq(networkInterfaceId)))
+                .thenReturn(networkInterfaceExpected);
+        PowerMockito.spy(AzureNetworkSDK.class);
+        PowerMockito.doReturn(networkInterfacesObject)
+                .when(AzureNetworkSDK.class, "getNetworkInterfaces", Mockito.eq(azure));
+
+        // exercise
+        Optional<NetworkInterface> networkInterface =
+                AzureNetworkSDK.getNetworkInterface(azure, networkInterfaceId);
+
+        // verify
+        Assert.assertTrue(networkInterface.isPresent());
+        Assert.assertEquals(networkInterfaceExpected, networkInterface.get());
+    }
+
+    // test case: When calling the getNetworkInterface method and do not find a network interface,
+    // it must verify if It returns a Optional with a network interface.
```

Comment:
The test case is misspelled

**Current:** `// it must verify if It returns a Optional with a network interface.`
**Right:** `// it must verify if It returns a Optional without a network interface.`

Reply:
Done.
I've made the change in the branch azure-compute-plugin-four-pr-fixes